rest-service/manager_rest/resource_manager.py

```diff
@@ -1272,14 +1272,34 @@ def cancel_execution(self, execution_ids, force=False, kill=False):
         :rtype: models.Execution
         :raises manager_exceptions.IllegalActionError
         """
-        if kill:
-            force = True
-        if kill:
-            new_status = ExecutionState.KILL_CANCELLING
-        elif force:
-            new_status = ExecutionState.FORCE_CANCELLING
-        else:
-            new_status = ExecutionState.CANCELLING
+        def _new_status_force(_force: bool, _kill: bool) -> (str, bool):
```

Comment:
I'm not convinced the closure makes things more readable here compared to just cleaning up the if statements- specifically, just removing the second 'if kill' would probably make the entire thing clean?